### PR TITLE
Single Page Application consisting of three navigation sections

### PR DESCRIPTION
**Pull request by ehs9nino**
Hi! We kindly ask you for a review of this project. We've completed this milestone where we were required to: Use JavaScript classes instead of objects and arrays. Use objects and arrays, but inside classes. And then we created single page application Create class methods to add and remove books. Maintain all of the original functionality of the application. Use only plain JavaScript. CSS styles to the application to make it match this wire frame.

We fulfilled the following requirements: 
        A Navigation bar.
        Three content sections:
            Books list.
            Add book form.
            Contact info.

Please let us know about any recommendations!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    day: 'numeric',
+    year: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+    hour12: 'true',
+  };
+  siteDate.textContent = `${date.toLocaleTimeString(locale, options)}`;
+  return siteDate;
+}
```
Comment (sja-thedude) [inline index.js:142]: - [x] Kindly make sure to display the current date using [Luxon](https://moment.github.io/luxon/#/install) as per the project requirements.
See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time).
- [x] Also make sure to align the date/time to the right as shown in the attached video as per the project requirements.

https://user-images.githubusercontent.com/70269507/165714564-a51ca3e3-e2a6-4db8-9a48-599a89e102a8.mov

File: README.md [hunk 2]
```diff
@@ -2,72 +2,67 @@
 
 # Project Name
 
-> Description the project.
+> Awesome books - Javascript Single Page Application
```
Comment (sja-thedude) [inline README.md:5]: - [x] Kindly make sure to add a description of the project in about 3- 4 sentences like "`In this project, we have created a fully functional website for Awesome books where we have added three sections: Books list, Add book form, a new Contact info section with a Navigation bar, and a Footer.`"

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi Team, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Great job on implementing the SPA according to the wireframe provided.
✔️ All linters are passing
✔️ Correct git-flow
✔️ Good PR title and description
✔️ Professional Readme

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (ppmarq1) [reply README.md:5]: resolved

File: index.js (hunk 1 above)
Comment (ppmarq1) [reply index.js:142]: resolved

**Comment by sja-thedude** [comment]
For the next Code reviewer: **Kindly consider the luxon requirement as Optional for the student as it's not mentioned on their side.**

**Review by elyor-doniyorov — APPROVED** [review]
Hi @ehs9nino and @ppmarq1, 

In my opinion, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by ehs9nino** [comment]
@elyor-doniyorov @sja-thedude  thanks for reviewing our work!

**Pull request merged**